lib/colors/__init__.py

```diff
@@ -952,10 +952,10 @@ def __init__(self, *args, **kwargs):
                         self.append(color(clr))
 
             # From a string (image/name/context).
-            if isinstance(arg, (str, unicode)):
+            if isinstance(arg, str):
                 if os.path.exists(arg):
                     n = 10
-                    if "n" in kwargs.keys(): n = kwargs["n"]
+                    if "n" in list(kwargs.keys()): n = kwargs["n"]
```

Comment:
It doesn't look like this needs to be wrapped in a list (or the next two times either)